internal/nginx/manager.go

```diff
@@ -271,6 +270,7 @@ func (lm *LocalManager) Start(done chan error) {
 // Reload reloads NGINX.
 func (lm *LocalManager) Reload() error {
 	// write a new config version
+	lm.metricsCollector.UpdateWorkerProcessCount("old")
```

Comment:
What about storing the metric float in the collector struct
```
type LocalManagerMetricsCollector struct {
	// Metrics
	reloadsTotal     prometheus.Counter
	reloadsError     prometheus.Counter
	lastReloadStatus prometheus.Gauge
	lastReloadTime   prometheus.Gauge
	processTotal     *prometheus.GaugeVec
         workerProcessesValue float64
} 
```

This way, we don't need to call this twice, but we can always manually set the `old` at the beginning of the `UpdateWorkerProcessCount` method, then calculate the new one, and update the `workerProcessesValue` of the struct with the new value.

This way we avoid calling this twice. The reason is that I/O operations like opening files are not efficient, so if we can reduce them that'll be nice.

Let me know what you think